harvard-lts/CURIOSity
# Adding helper text to PDS download modal.

**Adding helper text to PDS download modal.**
* * *

**JIRA Ticket**: [LTSMAINT-766](https://jira.huit.harvard.edu/browse/LTSMAINT-766)

# What does this Pull Request do?
This adds helper text to the "Download This Item" modal window for items that are PDS.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild the application from the `LTSMAINT-766` branch
* Go to an item that is delivered through PDS. One example is https://localhost:21407/demo/catalog/47-990006049270203941_RADSCHL:1267673
* Click on the "Download This Item" button.
* Confirm the message appears at the bottom of the modal.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
